src/run_common.cpp

```diff
@@ -67,6 +73,10 @@ common_run_option_values get_common_run_options(
     if (args.count("real-time")) {
         values.rtf_target = args["real-time"].as<double>();
     }
+    auto worker_threads = args["worker-threads"].as<int>();
```

Comment:
I *think* (but haven't verified) that this will crash if the user does not specify the `--worker-threads` option. See my comment about `default_value` vs. `implicit_value`.